docs/DeveloperGuide.md

```diff
@@ -391,7 +478,31 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     * 4a2. The current client's profile will be displayed.
 
       Use case ends
+      
+      
+**Use case: Create a Session**
+
+**MSS**
+1.  User requests to add a specific Session in the list
+2.  FitEgo adds the Session.
 
+    Use case ends.
+    
+**Extensions**
+
+* 1a. The session overlaps with an existing Session.
+    
+    * 1a1. FitEgo shows an error message.
+
+      Use case ends.
+      
+* 1b. The Session is missing some required details.
```

Comment:
Would this mean we need to have this extension in every command? (extension if we have a missing argument)

Reply:
should we?